### PR TITLE
Laravel 11.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 11.x.

**Before merging**, you need to:

- Checkout the `l11-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: Shift detected GitHub Actions which run jobs using a version `matrix`. Shift attempted to [update your configuration](https://github.com/Flowframe/laravel-trend/pull/53/commits/d2b0ce64a7916ddd97b612af42664e25975944aa?w=1) for Laravel 11. However, you should review these changes to ensure the desired combination of versions are built for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/laravel-trend.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "flowframe/laravel-trend": "dev-l11-compatibility",
    }
}
```

Finally, run: `composer update`

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -2,19 +2,22 @@ name: run-tests
...
   test:
     runs-on: ${{ matrix.os }}
+
     strategy:
       fail-fast: true
       matrix:
         os: [ubuntu-latest, windows-latest]
-        php: [8.1, 8.0]
-        laravel: [^8.71, 9.*, 10.*]
+        php: [8.0, 8.1, '8.2']
```
Comment (lloricode) [inline .github/workflows/run-tests.yml:19]: please add php8.3, thank you

**Review by lloricode — CHANGES_REQUESTED** [review]
please add php8.3, thank you

**Comment by ahinkle** [comment]
ping @Larsklopstra ❤️

**Comment by niclas-timm** [comment]
@Larsklopstra would be great to get this PR merged! Let me know if you need hep with anything.

**Comment by ConnorHowell** [comment]
Why not remove `illuminate/contracts` entirely? It's not used as far as I can tell (unless I'm missing something obvious?).

**Pull request merged**